priv/templates/phx.gen.live/form.ex

```diff
@@ -0,0 +1,98 @@
+defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>Live.Form do
+  use <%= inspect context.web_module %>, :live_view
+
+  alias <%= inspect context.module %>
+  alias <%= inspect schema.module %>
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div>
+      <.header>
+        <%%= @page_title %>
+        <:subtitle>Use this form to manage <%= schema.singular %> records in your database.</:subtitle>
+      </.header>
+
+      <.simple_form for={@form} id="<%= schema.singular %>-form" phx-change="validate" phx-submit="save">
+<%= Mix.Tasks.Phx.Gen.Html.indent_inputs(inputs, 8) %>
+        <:actions>
+          <.button phx-disable-with="Saving...">Save <%= schema.human_singular %></.button>
+        </:actions>
+      </.simple_form>
+
+      <.back navigate={return_path(@return_to.key, @<%= schema.singular %>)}>Back to <%%= @return_to.name %></.back>
```

Comment:
```suggestion
      <.back navigate={return_path(@return_to.key, @<%= schema.singular %>)}>Back</.back>
```

And then we skip the whole plural/singular? Or we make this generic and not related to the back functionality.


```suggestion      
      <%= if @<%= schema.singular %>.id %>
        <.link navigate={~p"<%= schema.route_prefix %>/#{@<%= schema.singular %>}"}>See <%= schema.singular %></.link> |
      <% end %>
      <.link navigate={~p"<%= schema.route_prefix %>"}>All <%= schema.plural %></.link>
```

Then we kill the <.back> in show as well and get rid of another component. WDYT @chrismccord?

Reply:
Actually, if we kill `.back`, we can do it another PR, to keep this one focused.